pkg/kritis/apis/kritis/v1beta1/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,133 @@
+// +build !ignore_autogenerated
```

Comment:
If you add a .gitattributes file to the top level, you can make these be ignored in github diffs:

```

# This file is documented at https://git-scm.com/docs/gitattributes.
# Linguist-specific attributes are documented at
# https://github.com/github/linguist.

**/zz_generated.*.go linguist-generated=true
/pkg/client/** linguist-generated=true
```

Reply:
awesome, done!